backend/app/main.py

```
@@ -1678,3 +1798,86 @@ async def delete_db_row(table_name: str, row_id: int, admin: dict = Depends(auth
         return RowMutationResponse(success=True, id=row_id)
     except Exception as e:
         return RowMutationResponse(success=False, error=str(e))
+
+
+@app.get("/admin/database/export")
+async def export_database(background_tasks: BackgroundTasks, _admin: Dict = Depends(auth.require_admin)) -> FileResponse:
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)